interfaces/builtin/opengl.go

```diff
@@ -183,6 +183,48 @@ unix (bind,listen) type=seqpacket addr="@cuda-uvmfd-[0-9a-f]*",
 # From https://bugs.launchpad.net/snapd/+bug/1862832
 /run/nvidia-xdriver-* rw,
 unix (send, receive) type=dgram peer=(addr="@var/run/nvidia-xdriver-*"),
+
+# Hybris support
+/{,var/}run/shm/hybris_shm_data rw, # FIXME: LP: #1226569 (make app-specific)
+# This is a LD_PRELOADed TLS padding library allowing use of both
+# Android 9 and glibc TLS slots at the same time. Only used on Halium 9.
+/usr/lib/@{multiarch}/libtls-padding.so  mr,
+/android{,/**} r,
+/{,android/}system/build.prop r,
+/{,android/}vendor/lib{,64}/**           r,
+/{,android/}vendor/lib{,64}/**.so        m,
+/{,android/}system/lib{,64}/**           r,
+/{,android/}system/lib{,64}/**.so        m,
+/{,android/}system/vendor/lib{,64}/**    r,
+/{,android/}system/vendor/lib{,64}/**.so m,
+/{,android/}odm/lib{,64}/**    r,
+/{,android/}odm/lib{,64}/**.so m,
+/{,android/}apex/com.android.runtime/lib{,64}/**     r,
+/{,android/}apex/com.android.runtime/lib{,64}/**.so  m,
+/{,android/}apex/com.android.i18n/lib{,64}/**.so     r,
+/{,android/}apex/com.android.i18n/lib{,64}/**.so     m,
+/{,dev/}socket/property_service rw, # attach_disconnected path
+/{,dev/}socket/logdw rw, # attach_disconnected path
+/{,dev/}__properties__/** rw, # attach_disconnected path
```

Comment:
Curious, where is __properties__ coming from? Is this some Android-specific interface? What's inside?

Reply:
The `__properties__` part contains SELinux info of process labels allowed to access Android properties through `getprop` and `setprop`. It's more of a void on UT though since functionality tied to SELinux is stubbed out to not conflict with AppArmor.

Reply:
Does it need `rw` access or just read?

Reply:
Potentially only needs read-access, since that is set up in the Halium container. But I'll have to check.

Reply:
Addressed